openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -125,17 +138,35 @@ def _update_tool_with_data(self, tool, data):
         original_subset = tool.GetData("openpype.subset")
         subset = data["subset"]
         if original_subset != subset:
-            # Subset change detected
-            # Update output filepath
-            workdir = os.path.normpath(legacy_io.Session["AVALON_WORKDIR"])
-            filename = f"{subset}..exr"
-            filepath = os.path.join(workdir, "render", subset, filename)
-            tool["Clip"] = filepath
-
-            # Rename tool
-            if tool.Name != subset:
-                print(f"Renaming {tool.Name} -> {subset}")
-                tool.SetAttrs({"TOOLS_Name": subset})
+            self._configure_saver_tool(data, tool, subset)
+
+    def _configure_saver_tool(self, data, tool, subset):
+        formatting_data = deepcopy(data)
+
+        # get frame padding from anatomy templates
+        anatomy = Anatomy()
+        frame_padding = int(
+            anatomy.templates["render"].get("frame_padding", 4)
+        )
+
+        # Subset change detected
+        workdir = os.path.normpath(legacy_io.Session["AVALON_WORKDIR"])
+        formatting_data.update({
+            "workdir": workdir.replace("\\", "/"),
+            "frame": "0" * frame_padding,
+            "ext": "exr"
+        })
+
+        # build file path to render
+        filepath = self.temp_rendering_path_template.format(
+            **formatting_data)
+
+        tool["Clip"] = filepath
+
+        # Rename tool
+        if tool.Name != subset:
+            print(f"Renaming {tool.Name} -> {subset}")
+            tool.SetAttrs({"TOOLS_Name": subset})
```

Comment:
```suggestion
            tool.SetAttrs({
                "TOOLB_NameSet": True,
                "TOOLS_Name": subset
            })
```
This fixes the issue described [here](https://github.com/ynput/OpenPype/pull/4943#issuecomment-1542176385) however it sets it without the extension which is something @EmberLightVFX mentioned he does like visually.

Reply:
One could add `_{ext}` after the name. Not the most beautiful but would show the extension in the name.

Reply:
@jakubjezek001 any preference on this?

Reply:
I agree with @EmberLightVFX that it would be better to keep the name with extension.